web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntity.java

```diff
@@ -15,27 +15,86 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.futurewei.alcor.web.entity.port.PortEntity;
 import com.futurewei.alcor.web.entity.route.RouteEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
 import lombok.Data;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
 
 @Data
 public class InternalPortEntity extends PortEntity {
 
-    @JsonProperty("routes")
-    private List<RouteEntity> routes;
+  @JsonProperty("routes")
+  private List<RouteEntity> routes;
 
-    @JsonProperty("neighbor_info")
-    private List<NeighborInfo> neighborInfos;
+  @JsonProperty("neighbor_info")
+  private List<NeighborInfo> neighborInfos;
 
-    @JsonProperty("binding_host_ip")
-    private String bindingHostIP;
+  @JsonProperty("binding_host_ip")
+  private String bindingHostIP;
```

Comment:
I think we can safely remove bindingHostIP as it is already included in PortEntity.

Reply:
no, PortEntity do not have the property at this time

Reply:
You are right. PortEntity has host id, but not host ip.